protoc-gen-grpc-gateway/main.go

```diff
@@ -28,7 +28,7 @@ var (
 	allowDeleteBody            = flag.Bool("allow_delete_body", false, "unless set, HTTP DELETE methods may not have a body")
 	grpcAPIConfiguration       = flag.String("grpc_api_configuration", "", "path to gRPC API Configuration in YAML format")
 	pathType                   = flag.String("paths", "", "specifies how the paths of generated files are structured")
-	modulePath                 = flag.String("module", "", "specifies a module prefix that will be stripped from the go package to determine the output directory")
+	_                          = flag.String("module", "", "specifies a module prefix that will be stripped from the go package to determine the output directory")
```

Comment:
I understand this flag is handled separately by `protogen`, could we just remove it instead of leaving it in like this? Is the `protogen` flag different? What other flags do you think we may need to worry about having duplicated, if any?

Reply:
Actually, `protogen` [uses](https://github.com/protocolbuffers/protobuf-go/blob/f62736dede0a74248d862247dbcb53ab69582a05/compiler/protogen/protogen.go#L180) the flag `module` but do not declare it.
Removing this line result in a failure:
> Cannot set flag module=github.com/yourorg/yourproject
> --grpc-gateway_out: protoc-gen-grpc-gateway: Plugin failed with status code 255

Reply:
Ha, weird. Alright, I'm happy to merge this now, I think we're going to be changing this part of the code soonish anyway.